tools/docker/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN : \
     make \
     ninja-build \
     python3 \
+    python3-pip \
```

Comment:
Can you please explain where is this necessary? 

All ESP-IDF related work will be done in the virtual environment created by `python3-venv`. `pip` is installed there so I think installing `python3-pip` is unnecessary. Moreover, it can be even misleading. If the user will install packages globally with `python3-pip` then it won't be available in the ESP-IDF environment.